setup.py

```diff
@@ -32,5 +32,19 @@
     packages=["fsspec", "fsspec.implementations"],
     python_requires=">3.6",
     install_requires=open("requirements.txt").read().strip().split("\n"),
+    extras_require={
+        "aws": ["s3fs"],
```

Comment:
I would probably use the same names as the protocols, so "s3" for this one and "gcs" instead of "google"

Reply:
I wasn't sure what the best approach was there. I think matching it is fine even if that means there are some redundancies.